w3d_translator/utils.py

```diff
@@ -1,6 +1,13 @@
 from errors import TranslationError
 
 
+def split_str(string: str, parenthesis=True) -> list:
+    """Splits a string based on commas, removes parenthesis if in str"""
+    string = string.replace("(", "").replace(")", "")
```

Comment:
can you rely on the comma+space always existing? or should you split on comma alone then strip out any extra whitespace?

Reply:
Yeah I got caught with that one in the `story` branch, I'll make the same change here